docs/_user_guide/best_practices_optimization.md

```diff
@@ -1,306 +1,354 @@
-before running the interactive Python scripts that use `gurobipy` package. 
+The environment is now set up to run your Python scripts that import `gurobipy` on the interactive Yen nodes. Keep in mind that the `module load` command and virtual environment activation are only active in the current shell.
 
+!!! Important
+    You need to load the `gurobipy3` module and activate your `venv` environment every time you log in to the Yens before running the interactive Python scripts that use the `gurobipy` package.
 
-### Running Gurobi in R
-Similar to running Gurobi in Python, Gurobi R package is also installed and available system-wide to use on the Yens. 
-You do not need to install anything into your user R library. 
+#### Running Gurobi in R
 
-**Environment Setup**
+Similar to running Gurobi in Python, Gurobi R package is also installed and available system-wide to use on the Yens. You do not need to install anything into your user R library.
 
-To use Gurobi software with R, you simply need to load both modules:
+##### Environment Setup
 
-```bash
-$ ml gurobi R
+To use Gurobi software with R, simply load both modules:
+
+```title="Terminal Command"
+ml gurobi R/4.2
 ```
 
-List loaded modules:
-```bash
-$ ml 
+To list the currently loaded modules, use:
+
+```title="Terminal Command"
+ ml
+```
+```{ .yaml .no-copy title="Terminal Output" }
 Currently Loaded Modules:
-  1) gurobi/10.0.0   2) rstudio/2022.07.2+576   3) R/4.2.1
+  1) gurobi/10.0.0   2) R/4.2.0
 ```
 
 Launch interactive R:
-```bash
-$ R
+
+```title="Terminal Command"
+ R
 ```
 
 Then you can load the `gurobi` R package:
 
-```R
-R
-
-R version 4.2.1 (2022-06-23) -- "Funny-Looking Kid"
-Copyright (C) 2022 The R Foundation for Statistical Computing
+```{ .yaml .no-copy title="Terminal Output" }
+R version 4.3.0 (2023-04-21) -- "Already Tomorrow"
+Copyright (C) 2023 The R Foundation for Statistical Computing
 Platform: x86_64-pc-linux-gnu (64-bit)
 
 R is free software and comes with ABSOLUTELY NO WARRANTY.
 You are welcome to redistribute it under certain conditions.
 Type 'license()' or 'licence()' for distribution details.
 
-  Natural language support but running in an English locale
-
 R is a collaborative project with many contributors.
 Type 'contributors()' for more information and
 'citation()' on how to cite R or R packages in publications.
 
 Type 'demo()' for some demos, 'help()' for on-line help, or
 'help.start()' for an HTML browser interface to help.
+
 Type 'q()' to quit R.
+>
+```
 
-> library('gurobi')
+```title="Terminal Command"
+library('gurobi')
+```
+```{ .yaml .no-copy title="Terminal Output" }
 Loading required package: slam
 >
 ```
 
 You can now run the R scripts to solve the optimization problem using Gurobi on interactive Yen nodes.
 
-**Important:** You need to load the `gurobi` and `R` modules every time you login to the Yens
-before running the interactive R scripts that use `gurobi` R package.
+!!! Warning
```

Comment:
I would move the warning right after Running Gurobi in R title so folks can load the right R module